src/pages/index.tsx

```diff
@@ -20,38 +20,68 @@ export default function Home() {
       </Head>
       <Box
         sx={{
-          position: smallScreen ? undefined : "sticky",
+          position: "sticky",
           top: 0,
           backgroundColor: "#121212",
           zIndex: 100,
           mb: 4,
+          flexDirection: "row",
+          display: "flex",
+          alignItems: "center",
         }}
       >
-        <Typography
-          variant="h1"
-          sx={{ textAlign: "center", fontFamily: inter }}
-        >
-          <b>League Streaming Platform</b>
-        </Typography>
-        <Divider
-          sx={{
-            my: 1,
+        <img
+          src="/logo.png"
+          alt="League Streaming Platform"
+          width="100"
+          height="100"
+          style={{
+            marginLeft: 15,
+            marginTop: 15,
+            borderRadius: 30,
+            marginBottom: 15,
           }}
         />
         <Typography
-          variant="h3"
-          sx={{ textAlign: "center", fontFamily: inter }}
+          variant={smallScreen ? "h6" : "h1"}
+          sx={{
+            fontFamily: inter,
+            ml: 2,
+            color: "white",
+          }}
         >
-          <b>Season 1: F1 Twitter Refugees league</b>
+          League Streaming Platform
```

Comment:
Text not centered anymore

Reply:
That's on purpose in case we want an 3 dot menu at the left side, or some button next to the text. Also looks cleaner on mobile now.

Reply:
Yes, but it really dont looks nice on computer